ci-operator/step-registry/metadata.json

```diff
@@ -1548,6 +1548,23 @@
 				]
 			}
 		},
+		"openshift-upgrade-gcp-loki-workflow.yaml": {
```

Comment:
This file means DPTP needs to sign off on all new workflows or owner changes, right?  Is that intentional?

Reply:
I don't see a simple way around it :(

Reply:
Unless we put the metadata into each dir? idk.

Reply:
That comes from OWNERS file, which is a symlink to `step-registry/openshift/OWNERS`. I think OTA team should own `openshift/upgrade` - we can fix that in a followup PR, does that sound good?

Reply:
I'm ok with OTA-update folks owning the upgrade steps/workflows.  But that's orthogonal to this file, which is copying data from owned steps up to a parent directory with a different owner set.

Putting the metadata in each directory sounds good to me.  I'm not clear on why we need the metadata files at all, vs. having the consuming tool read `OWNERS` directly...

Reply:
doesn't have to get sorted this PR either ;)